### ISSUE TITLE
SkyNet training

### ISSUE THREAD

**Issue by dujuan1999** [TomG008/SkyNet#4]
Hello!I am a novice, I would like to ask if I can use my own dataset to do target detection training with skynet?I can't seem to find the code.
Thanks!

**Comment by TomG008** [comment]
@dujuan1999 Yes. You could use your own datasets. For object detection task, please refer to these [training scripts](https://github.com/onioncc/iSmartDNN/tree/master/DNN_train). For tracking, you could refer to SkyNet/Tracking/tools/ in this repo.

**Comment by deepseek** [comment]
@TomG008 is the iSmartDNN repository to train gpu or FPGA?

**Comment by TomG008** [comment]
@deepseek In iSmartDNN repo, DNN training is done by GPU. After training, you could import your model to different devices (GPU, FPGA, CPU) for inference.

**Comment by DeyanLan** [comment]
> @deepseek In iSmartDNN repo, DNN training is done by GPU. After training, you could import your model to different devices (GPU, FPGA, CPU) for inference.

Hello,  I want to train a new network for object detection task. I found that it isn't NAS but  a fixed  network structure in iSmartDNN repo. Could u please provide the related source-code for the CNN network searching project? 
If convenience, you could contact me via the CAS-email : landeyan@sia.cn

**Comment by SumantSakhalkar** [comment]
Hey @DeyanLan, thanks for the question. I had the same problem. I too was looking for NAS instead of the fixed n/w. Did you get any updates from the author? Any help would be appreciated.
